airbyte-scheduler/app/src/main/java/io/airbyte/scheduler/app/SchedulerApp.java

```diff
@@ -286,7 +283,7 @@ public static void main(final String[] args) throws IOException, InterruptedExce
           configRepository,
           new WorkspaceHelper(configRepository, jobPersistence),
           TrackingClientSingleton.get());
-      final TemporalClient temporalClient = TemporalClient.production(temporalHost, workspaceRoot, configs);
+      final TemporalClient temporalClient = TemporalClient.production(configs);
```

Comment:
Small refactor to let `TemporalClient.production` pull the `temporalHost` and `workspaceRoot` from the `configs` that we already pass in.